### DISCUSSION TITLE
Linux Osu!Lazer Backround Video missing

### DISCUSSION THREAD

**Discussion by bearrrmann** [ppy/osu#16039] [Q&A] [answered]
So I love osu lazer and ive used it for a while, but i was forced to use wine on linux because while the osu lazer login worked great, for some reason I tried to fix it the backround video on lazer wouldnt play on any of my maps, the ones I KNEW had backround video, it worked on windows but when I switched to linux the backround video doesnt work anymore, I dont know what to do, can you help?
summary- no backround video for kubuntu appimage osu lazer

**Comment by peppy** [comment, marked as answer]
on linux you need to mange your own video playback libraries. please make sure you have ffmpeg setup (use google), or if this isn't your kind of thing, you may want to consider running on windows instead.
